src/protocol.h

```diff
@@ -265,7 +265,17 @@ extern const char* WTXIDRELAY;
 }; // namespace NetMsgType
 
 /* Get a vector of all valid message types (see above) */
-const std::vector<std::string>& getAllNetMessageTypes();
+const std::map<uint8_t, std::string>& getAllNetMessageTypes();
+
+/** Short message type IDs are a low bandwidth representations of a message type
```

Comment:
d47be87: totally ignorable yocto nits for whenever you force push next ;)
```suggestion
/** Short message type IDs are low bandwidth representations of a message type
```